packages/desktop/src/renderer/components/Channel/File/FileComponent/FileComponent.test.tsx

```diff
@@ -38,13 +38,14 @@ describe('FileComponent', () => {
           downloadState: DownloadState.Ready,
           downloadProgress: undefined,
         }}
+        isUnsent={false}
```

Comment:
I think we are missing components tests for case where `isUnsent={true}`.

Reply:
Added component tests